dashboard/api.py

```diff
@@ -331,6 +336,32 @@ def format_courserun_for_dashboard(course_run, status_for_user, certificate=None
     return formatted_run
 
 
+def _check_if_refresh(user, cached_model, refresh_delta):
+    """
+    Helper function to check if cached data in a model need to be refreshed.
+    Args:
+        user (django.contrib.auth.models.User): A user
+        cached_model (dashboard.models.CachedEdxInfoModel): a model containing cached data
+        refresh_delta (datetime.datetime): time limit for refresh the data
```

Comment:
Nevermind, I see that it's a `datetime`. Can we pass in a `timedelta` instead of having the caller calculate `now - delta`?